httpgrpc/protocol_versions.go

```diff
@@ -29,4 +31,19 @@ package httpgrpc
 const (
 	UnaryRpcContentType_V1  = "application/x-protobuf"
 	StreamRpcContentType_V1 = "application/x-httpgrpc-proto+v1"
+
+	GrpcWebJsonContentType_V1 = "application/grpc-web+json"
```

Comment:
I realized that we've never even tried to test this with an actual grpc-web client. So it may be premature to add this here since it's possible this server wouldn't actually even work correctly. (I know making use of this content-type was my idea originally; sorry to back-track.)

Reply:
Hmm, we should probably try it out and see if it works I guess?